### ISSUE TITLE
Exporting as data/URL failed in live website

### ISSUE THREAD

**Issue by anioncat** [automatarium/automatarium#453]
### Description of the bug

User was unable to import a project they exported to themselves.

This didn't work in testing and wasn't working until halfway through writing this issue. The tester used to same machine as the one I am using on the same day with the same browser. It is working now but please keep a lookout for it. I don't know what happened.

### To reproduce

Steps the user took:
1. Go to [live website](https://automatarium.tdib.xyz/new)
2. Open/create any project, preferable add a state
3. Go to File -> Export -> Export as URL/Raw Data...
4. Copy either (link is easier)
5. Return to the projects page
6. Delete the shared project
7. Import the project

### Expected behavior

I get the project that I exported.

**Comment by s3897720** [comment]
Can confirm exporting from Firefox (123.0) to Edge (124.0.2478.51) works fine, and exporting from the same versions of Edge to Firefox also works fine. Is it a specific machine issue? If so, do you still have that machine?

**Comment by anioncat** [comment]
I've installed a different OS on the machine so I wouldn't say I have the same machine anymore. I should've mentioned more clearly in the issue that it never happened again with the same system (Windows) with the same browser (whatever version of Firefox at the time).

There were no other reports of it happening for anyone else, but this was a thing that happened during user testing so I had to document it somehow in case it cropped up elsewhere.
